Sources/KeyboardSupport/KeyboardScrollable.swift

```diff
@@ -156,6 +156,10 @@ public extension KeyboardScrollable where Self: UIViewController {
         } else {
             mutableInset.bottom += keyboardHeight
         }
+
+        let scrollViewConvertedFrame = view.window?.convert(scrollView.frame, from: view) ?? .zero
```

Comment:
I think this assumes the `scrollView` is a direct subview of the view controller. If so, do we want to make that assumption, and if so if so, should we document that assumption in the code?

I think we could also do `view.window?.convert(scrollView.frame, from: scrollView.superview) ?? .zero` to protect against that.

Reply:
Ah yeah good catch. I'll update